### PR TITLE
[PT-BR] translate getting-started the-console-module

### PR DESCRIPTION
**Pull request by JCMais**

### CODE REVIEW

**Comment by JCMais** [comment]
@nodejs/nodejs-pt review please

File: locale/pt-br/knowledge/getting-started/the-console-module.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+date: '2011-08-26T10:08:50.000Z'
+tags:
+  - core
+  - cli
+  - globals
+difficulty: 1
+layout: knowledge-post.hbs
+---
+
+Qualquer pessoa familiar com o desenvolvimento voltado para navegadores provavelmente já usou `console.log` para fins de debug - Node.js implementou um objeto próprio `console` para imitar grande parte desta experiência. Porém, como nós estamos trabalhado do lado do servidor, ela funciona como uma camada acima do `stdout`, `stdin` e `stderr` em vez do console de debug do navegador.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/the-console-module.md]:
```suggestion
Qualquer pessoa familiar com o desenvolvimento voltado para navegadores provavelmente já usou `console.log` para fins de debug - O Node.js implementou um objeto `console` próprio para imitar grande parte desta experiência. Porém, como nós estamos trabalhado do lado do servidor, ela funciona como uma camada acima do `stdout`, `stdin` e `stderr` em vez do console de debug do navegador.
```

File: locale/pt-br/knowledge/getting-started/the-console-module.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Por conta deste paralelo com o navegador, o módulo `console` se tornou a casa para grande parte da funcionalidade de "output" do Node. O mais simples é `console.log()`.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/the-console-module.md]:
```suggestion
Por conta deste paralelo com o navegador, o módulo `console` se tornou a principal API para grande parte da funcionalidade de "output" do Node. O mais simples é o `console.log()`.
```

File: locale/pt-br/knowledge/getting-started/the-console-module.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+const name = 'Harry';
+const number = 17;
+const myObj = {
+  propOne: 'alguma coisa',
+  propTwo: 'outras coisas'
+};
+console.log('Meu nome é %s, meu número é %d, e o objeto é %j', name, number, myObj);
+```
+
+Uma pegadinha com a função `console.log`, e todas as funções que dependem nela, é que ela possui um "buffer" para a sua saída. Portanto se o processo é interrompido abruptamente, seja por meio de algum erro ou através do `process.exit()`, é bem possível que o conteúdo no buffer nunca seja definitivamente enviado para a tela. Isso pode causar certa frustação, portanto fique atento para esta situação.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/the-console-module.md]:
```suggestion
Uma pegadinha com a função `console.log`, e todas as funções que dependem dela, é que ela possui um "buffer" para a sua saída. Portanto se o processo é interrompido abruptamente, seja por meio de algum erro ou através do `process.exit()`, é bem possível que o conteúdo no buffer nunca seja definitivamente enviado para a tela. Isso pode causar certa frustração, portanto fique atento para esta situação.
```

File: locale/pt-br/knowledge/getting-started/the-console-module.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+`console.error()` funciona do mesmo modo que `console.log`, exceto pelo fato que o conteúdo enviado para `stderr` em vez de `stdout`. Isso é na realidade uma diferença extremamente importante, pois o conteúdo enviado para `stderr` é sempre escrito de forma síncrona. Qualquer uso de `console.error`, ou qualquer outra função no core do Node.js que escreva para `stderr`, irá bloquear o processo até que todo o conteúdo seja escrito. Isso é útil para mensagens de error - você recebe elas no momento que elas acontecem - porém se usado de forma indiscriminada, pode tornar o processo do Node.js bastante lento.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/the-console-module.md]:
```suggestion
`console.error()` funciona do mesmo modo que `console.log`, exceto pelo fato que o conteúdo enviado para `stderr` em vez de `stdout`. Isso é, na realidade, uma diferença extremamente importante, pois o conteúdo enviado para `stderr` é sempre escrito de forma síncrona. Qualquer uso de `console.error`, ou qualquer outra função no core do Node.js que escreva para `stderr`, irá bloquear o processo até que todo o conteúdo seja escrito. Isso é útil para mensagens de erro - você as recebe no momento que elas acontecem - porém se usado de forma indiscriminada, pode tornar o processo do Node.js bastante lento.
```

File: locale/pt-br/knowledge/getting-started/the-console-module.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+`console.dir()`, como mencionado acima, é um "alias" para `util.inspect()` - é uma função usada para enumerar as propriedades de um objeto.  [Leia mais](/pt-br/knowledge/getting-started/how-to-use-util-inspect/)
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/the-console-module.md]:
```suggestion
`console.dir()`, como mencionado acima, é um "alias" para `util.inspect()` - é uma função usada para enumerar as propriedades de um objeto. [Leia mais](/pt-br/knowledge/getting-started/how-to-use-util-inspect/)
```

File: locale/pt-br/knowledge/getting-started/the-console-module.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+var string = '';
+for (var i = 0; i < 300; i++) {
+  (function (i) {
+    string += 'aaaa' + i.toString();
+  })(i);
+}
+console.timeEnd('timer');
+```
+
+Isto iria determinar a quantidade de tempo necessária para executar as ações entre as chamadas de `console.time` e `console.timeEnd`.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/the-console-module.md]:
```suggestion
Isto irá determinar a quantidade de tempo necessária para executar as ações entre as chamadas de `console.time` e `console.timeEnd`.
```

File: locale/pt-br/knowledge/getting-started/the-console-module.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+Uma última função que merece ser mencionada é `console.trace()`, que exibe o conteúdo da pilha (stack trace) do momento que é chamado, sem ocasionar um erro. Isso ocasionalmente pode vim a ser útil caso você queira descobrir de onde determinada função defeituosa foi chamada.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/the-console-module.md]:
```suggestion
Uma última função que merece ser mencionada é `console.trace()`, que exibe o conteúdo da pilha (stack trace) do momento que é chamado, sem ocasionar um erro. Isso ocasionalmente pode vir a ser útil caso você queira descobrir de onde determinada função com erro foi chamada.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Some minor changes

**Comment by JCMais** [comment]
Thanks @khaosdoctor

File: locale/pt-br/knowledge/getting-started/the-console-module.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+date: '2011-08-26T10:08:50.000Z'
+tags:
+  - core
+  - cli
+  - globals
+difficulty: 1
+layout: knowledge-post.hbs
+---
+
+Qualquer pessoa familiar com o desenvolvimento voltado para navegadores provavelmente já usou `console.log` para fins de debug - O Node.js implementou um objeto `console` próprio para imitar grande parte desta experiência. Porém, como nós estamos trabalhado do lado do servidor, ela funciona como uma camada acima do `stdout`, `stdin` e `stderr` em vez do console de debug do navegador.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/the-console-module.md]:
```suggestion
Qualquer pessoa familiarizada com o desenvolvimento voltado para navegadores provavelmente já usou `console.log` para fins de debug - O Node.js implementou um objeto `console` próprio para imitar grande parte desta experiência. Porém, como nós estamos trabalhado do lado do servidor, ela funciona como uma camada acima do `stdout`, `stdin` e `stderr` em vez do console de debug do navegador.
```

Didn't see it in my previous review

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Just a small thing I didn't see

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
@MaledongGit this is good to go!

**Pull request merged**